apps/cowswap-frontend/src/modules/tokens/hooks/useCurrencyBalance.ts

```diff
@@ -45,10 +48,13 @@ export function useNativeCurrencyBalances(uncheckedAddresses?: (string | undefin
       validAddressInputs.reduce<{ [address: string]: CurrencyAmount<Currency> }>((memo, [address], i) => {
         const value = results?.[i]?.result?.[0]
         if (value && chainId)
-          memo[address] = CurrencyAmount.fromRawAmount(nativeOnChain(chainId), JSBI.BigInt(value.toString()))
+          memo[lowerCaseAddress ? address.toLowerCase() : address] = CurrencyAmount.fromRawAmount(
```

Comment:
Why not make all addresses lowercase?

Reply:
I'm not sure about other cases, don't want to break smth